pkg/ali/README.md

```diff
@@ -0,0 +1 @@
+# Ali Cloud Provisioning
```

Comment:
We haven't done a good job of this everywhere, but i think this is the content we show in the slide out when browsing extensions. So it should contain a short description of what the plugin is and does